readme.md

```diff
@@ -1,3 +1,42 @@
 # Product Owner Guide
 
 A rough guide for those who are product owners on a dwyl project.
+
+## What is a Product Owner?
+
+**A Product Owner (PO)** is someone who represents the views and needs of the
+target audience, stakeholders and the business they represent in agile scrum
+methodology. They must understand and communicate the vision of the product to
+the rest of the team.
+
+They are the liaison between customers and the development team. This means that
+they are responsible for working with stakeholders and user groups to determine
+their needs. Then through collaboration with the Scrum Master and developer team
+they prioritise and discuss these ideas moulding them into features to be built.
+
+**PO Responsibilities:**
+1. Owner and creator of the product backlog* :crown:
+2. Adding new features to the backlog. :new:
+3. Approving or critiquing development work for release. :white_check_mark:
+4. Clearly define, understand and _empathise_ with the end-user. 👩 💡
+5. Involving customers or users and getting their feedback to continuously shape
+the product development. :+1: :thumbsdown:
+6. Maintaining and _updating_ the backlog: adjusting the features and _priority_
+as needed after each sprint or iteration. :small_red_triangle: :small_red_triangle_down:
+7. Describing the user-persona(s) in as much detail as possible to make the
+"pain" they feel _real_ (_especially useful when the team is remote and don't
+often meet with the end-users_) 📝
+8. Attach a _business_ or _customer_ value to each story/task in the backlog to
```

Comment:
This point might make more sense right next to the adding of new features to the backlog for example, but I don't know if you have already put these in an order that makes sense to you @Cleop ?

Not a blocker.